__tests__/app.test.js

```diff
@@ -108,3 +106,55 @@ describe("GET API/reviews/:review_id", () => {
       });
   });
 });
+
+describe("GET /api/reviews/:review_id/comments", () => {
+  test("should an array of comments for the given review_id of which each comment should have the given properties", () => {
```

Comment:
be aware of the test descriptions here, it doesnt read very well, I might have said "should respond with an array .... etc"